msteams-platform/concepts/device-capabilities/native-device-permissions.md

```diff
@@ -106,69 +100,85 @@ navigator.permissions.query({name:'geolocation'}).then(function(result) {
 });
 ```
 
-## Prompting the user
+## Use Teams APIs to get device permissions
 
-To show a prompt to get consent to access device permissions you need to leverage the appropriate HTML5 or Teams API. 
+Leverage appropriate HTML5 or Teams API, to display a prompt for getting consent to access device permissions.
 
-For example, to prompt the user to access their location you need to call `getCurrentPosition`:
+> [!IMPORTANT]
+> * Support for `camera`, `gallery`, and `microphone` is enabled through [**selectMedia API**](/javascript/api/@microsoft/teams-js/media?view=msteams-client-js-latest#selectMedia_MediaInputs___error__SdkError__attachments__Media_______void_&preserve-view=true). Use [**captureImage API**](/javascript/api/@microsoft/teams-js/microsoftteams?view=msteams-client-js-latest#captureimage--error--sdkerror--files--file-------void-&preserve-view=true) for a single image capture.
+> * Support for `location` is enabled through [**getLocation API**](/javascript/api/@microsoft/teams-js/location?view=msteams-client-js-latest#getLocation_LocationProps___error__SdkError__location__Location_____void_&preserve-view=true). You must use this `getLocation API` for location, as [**geolocation API**](../../resources/schema/manifest-schema.md#devicepermissions) is currently not fully supported on all desktop clients.
```

Comment:
Hyperlinks are giving 404 error. Please check again.

Reply:
The links will work once the PR is merged as these APIs are present in a different docset.